### PR TITLE
#cosmwasm dev tools to play with contracts

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**
## Issue

Started to move cosmwasm tools into flake-part.

Remove subxt from wasm tools (as it is not for sure such).

Add cosmwasm check.

### CODE REVIEW

**Review by KaiserKarel — APPROVED** [review]

File: code/xcvm/cosmos/flake-parts.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{ self, ... }: {
```
Comment (cor) [inline code/xcvm/cosmos/flake-parts.nix:1]: Please rename this file to `cosmwasm.nix` instead of `flake-parts.nix`

**Review by cor — CHANGES_REQUESTED** [review]
Looks good! Glad you're getting started with `flake-parts`. Only one nitpick, please rename the flake-parts file

File: code/xcvm/cosmos/flake-parts.nix (hunk 1 above)
Comment (dzmitry-lahoda) [reply code/xcvm/cosmos/flake-parts.nix:1]: https://github.com/hercules-ci/flake-parts/tree/main/template/multi-module/hello

Comment (dzmitry-lahoda) [reply code/xcvm/cosmos/flake-parts.nix]: renamed, hopefully according dicussion

**Review by cor — APPROVED** [review]
LGTM

**Pull request merged**